i18n/es.json

```diff
@@ -2622,6 +2630,18 @@
     "id": "app.import.validate_direct_post_import_data.user_missing.error",
     "translation": "Falta la propiedad obligatoria del mensaje del canal directo: user"
   },
+  {
+    "id": "app.import.validate_emoji_import_data.empty.error",
+    "translation": "Datos de importación del emoticon vacío."
+  },
+  {
+    "id": "app.import.validate_emoji_import_data.image_missing.error",
+    "translation": "El campo de imagén del emoticon en la importación está faltante o vacío."
```

Comment:
This sounds weird in Spanish from Spain. I would use here `El campo de imagén del emoticon en la importación está ausente o vacío.`.

Reply:
can you please go to the translation server and make the changes there ;)

thanks my friend.

Reply:
Sure

Reply:
Done :)